x/pricefeed/client/cli/tx.go

```diff
@@ -20,7 +29,45 @@ func GetTxCmd() *cobra.Command {
 		RunE:                       client.ValidateCmd,
 	}
 
-	// this line is used by starport scaffolding # 1
+	cmd.AddCommand(
+		CmdMintNft(),
+	)
```

Comment:
why `MintNft`?
isn't that supposed to be like `PostPrice`?

Reply:
@taiki1frsh 
I made mistake.
I've fixed it.
plz check it